inc/Admin.php

```diff
@@ -13,6 +13,15 @@
  * Admin class.
  */
 class Admin {
+
+	/**
+	 * Property to suspend the survey.
+	 * Change it to true to enable it.
+	 *
+	 * @var bool
+	 */
+	private $suspend_survey = false;
```

Comment:
Sorry for missing this initially, but shouldn't it be the other way around not to display the notice?

Reply:
Sorry for missing it, I've enabled it to test the behavior of other notices and forgot it like this.